xsuite/src/world/fsworld.ts

```diff
@@ -185,6 +203,27 @@ export class FSWorld extends World {
     return super.deployContract(tx).then((r) => this.addContractPostTx(r));
   }
 
+  async addKvs(address: AddressLike, kvs: EncodableKvs) {
```

Comment:
"async" not necessary

Let's add this rule in the eslint config in error mode:

https://eslint.org/docs/latest/rules/require-await

Reply:
#219 for the eslint rule